universe/base.go

```diff
@@ -236,14 +236,6 @@ func (a *MintingArchive) verifyIssuanceProof(ctx context.Context, id Identifier,
 		return nil, fmt.Errorf("asset id mismatch: expected %v, got %v",
 			id.AssetID, newAsset.ID())
 
-	// The outpoint of the final resting place of the asset should match
-	// the leaf key
-	//
-	// TODO(roasbeef): this restrict to issuance
-	case assetSnapshot.OutPoint != key.MintingOutpoint:
```

Comment:
+1 for removing here 

I think this is also where we'd add the logic to ensure we only get issuance proofs or not (depending on the future config).